docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+        * `/metadata/labels/new-label`
+        * `/metadata/annotations/new-annotation`
+        * `/spec/template/spec/containers/0/resources/limits/cpu`
+        * `/spec/template/spec/containers/0/resources/requests/memory`
+
+
+- `value`: The value to be set.
+    * If the `op` is `remove`, the value cannot be set.
+
+
+### Multiple Override Patches
+You may add multiple `JSONPatchOverride` to an `OverrideRule` to apply multiple changes to the selected cluster resources.
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: cro-1
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+          - op: remove
+            path: /rules/0/verbs/1
+```
+The `ClusterResourceOverride` object above will remove the verbs "list" and "watch" in the `ClusterRole` named 
+`secret-reader` on clusters with the label `env: prod`.
+
+## Applying the ClusterResourceOverride
+Create a ClusterResourcePlacement resource to specify the placement rules for distributing the cluster resource overrides across
+the cluster infrastructure. Ensure that you select the appropriate resource.
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacement
+metadata:
+  name: crp
+spec:
+  resourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    placementType: PickAll
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+```
+The `ClusterResourcePlacement` configuration outlined above will disperse resources across all clusters labeled with `env: prod`. 
+As the changes are implemented, the corresponding `ClusterResourceOverride` configurations will be applied to the 
+designated clusters, triggered by the selection of matching cluster role resource `secret-reader`.
+
+To ensure that the `ClusterResourceOverride` object is applied to the selected resources, verify the `ClusterResourcePlacement`
+status:
+```
+Status:
+  Conditions:
+    ...
+    Last Transition Time:   2024-04-27T04:18:00Z
+    Message:                The selected resources are successfully overridden in the 10 clusters
+    Observed Generation:    1
+    Reason:                 OverriddenSucceeded
+    Status:                 True
+    Type:                   ClusterResourcePlacementOverridden
+    ...
+  Observed Resource Index:  0
+  Placement Statuses:
+    Applicable Cluster Resource Overrides:

```

Comment:
why there are spaces between the field name?

Reply:
Okay, I see the difference if I use `kubectl describe crp`  it will show spaces for the different fields, but if `kubectl get crp crp-example -o yaml` it will show with no spaces. Is there a preference to the latter?

Reply:
oh, i see, you can probably change the 

verify the `ClusterResourcePlacement` by using `kubectl describe crp` command, 

my assumption is to use the kubectl get -o yaml command :)